### PR TITLE
Bump Devise to a version with Rails 6 support

### PR DESCRIPTION
**Pull request by koppen**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -178,7 +178,7 @@ GEM
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.0.4)
+    rails-html-sanitizer (1.2.0)
```
[the remark this answers is not in the document]
Comment (koppen) [reply Gemfile.lock]: According to https://www.cvedetails.com/cve/CVE-2015-7580/, this is solely an issue when running Rails 4.x or 5.0.x. So it should not be an actual issue for us in this instance.

[the remark this answers is not in the document]
Comment (koppen) [reply Gemfile.lock]: Also, it is version 1.0.2 that supposedly has this vulnerability.

[the remark this answers is not in the document]
Comment (olepalm) [reply Gemfile.lock]: Could this be due to Rails 6 being rather new, so the CVE record hasn't been updated? Not that we have much choice, if we need devise.

[the remark this answers is not in the document]
Comment (koppen) [reply Gemfile.lock]: It's fixed in [this commit](https://github.com/rails/rails-html-sanitizer/commit/63903b0eaa6d2a4e1c91bc86008256c4c8335e78) which is part of 1.2.0, as far as I can tell.

**Pull request merged**